config/locales/en.yml

```diff
@@ -429,4 +460,285 @@ en:
+      temporality_header: Temporality
+      format_results_as: "Format results as:"
+      reproduce_results_using: Reproduce these results using the
+      additional_parameters_explained: Additional parameters explained at
+      annotator_api_documentation: Annotator API documentation
+
+  schemes:
+    id: ID
+    preferred_name: Preferred Name
+    type: Type
+  fair_score:
+    mod_link: MOD1.4
+    see_the_used_properties: see the used properties
+    metadata_properties: Metadata properties listed in the following are picked from %{mod_link}.
+    ontology_repository: This ontology repository implements %{mod_link} but does not necessarily encode metadata with the same properties (%{submission_link}).
+    see_possible_credits: See possible credits
+    see_metadata_used_properties: See metadata used properties
+    not_found: not found
+    null: "null"
+  ontologies_metadata_curator:
+    ontologies: Ontologies
+    get_values: Get values
+    select_ontologies_and_metadata: select ontologies and the metadata properties
+    include_all_submissions: Include all submissions
+    apply_the_change_for_all: Apply the change for all
+    update_the_current_displayed_content: "will update the current displayed content to all the following submissions:"
+    save: Save
+  ontologies:
+    no_views: No views available for %{acronym}.
+    fairness_assessment_questions: O'FAIRe FAIRness assessment questions
+    see_details: See details
+    collecting_data: We are still collecting data for %{acronym}
+    location: Location
+    allow_users: Allow users to view the metadata of your ontology (Summary) only.
+    metadata_only: Metadata only (No file)
+    load_from_url: Load from URL
+    new_versions_loaded: New versions loaded on a nightly basis.
+    upload_local_file: Upload local file
+    ontology_submitted: Ontology submitted successfully!
+    submitting_metadata_only: Thank you for submitting your metadata-only ontology to %{site}.
+    users_can_see: Users can now see
+    your_ontology: your ontology
+    exploring_and_searching: | 
+      in our ontology list but they cannot explore or search it. To enable exploring and searching,
+      please upload a full version of your ontology.
+    submitting_ontology_view: Thank you for submitting your ontology view to %{site}.
+    submitting_ontology: Thank you for submitting your ontology to %{site}.
+    processing_message: |
+      We will now put your ontology in the queue to be processed. Please keep in mind that it may take up to several
+      hours before %{site} users will be able to explore and search your ontology.
+    notification_message: "When your ontology is ready for viewing, you will receive an email notification and it will be available here:"
+    contact_support: "If you have any questions or problems, please email the %{site} support team at:"
+    collecting_data_message: We are still collecting data for %{acronym}
+    submit_new_ontology: Submit New Ontology
```

Comment:
this need to be `Submit new ontology` not `Submit New Ontology` to pass the system tests